docs/using-getting-started.asciidoc

```diff
@@ -0,0 +1,304 @@
+[[ecs-getting-started]]
+=== Getting Started
+
+ECS enables and encourages users to normalize event data in order
+to better analyze, visualize, and correlate their events. Collected events
+can be normalized at ingest time, consistently searched across indices,
+and visualized predictably. When adopting an Elastic solution, such as
```

Comment:
To make sure this stands out more, I'd start a new paragraph at "When".

Perhaps rephrase "Note that when..."